remoter-client/src/wrapper.r

```diff
@@ -16,7 +16,7 @@ load_config <- function(development_aws_server) {
     )
 
     if(config$cluster_env == 'development') {
-        config$aws_config[['endpoint']] <- sprintf("http://%s:4566", development_aws_server) # DOCKER_GATEWAY_HOST
+        config$aws_config[['endpoint']] <- sprintf("http://192.168.43.241:4566", development_aws_server) # DOCKER_GATEWAY_HOST
```

Comment:
You forgot to remove the hard IP?

Reply:
......

Reply:
I actually thought I had already fixed this hahah